vm/internal/handler/export.go

```diff
@@ -70,9 +70,11 @@ func (h *Handler) ExportVolume(ctx echo.Context) error {
 	// Export
 	cmd := []string{
 		"tar",
-		"-zcvf",
-		"/vackup" + "/" + filepath.Base(fileName),
-		"/vackup-volume",
+		"zcvf",
```

Comment:
why did the `-` disappear before options ?

Reply:
Recently I learnt that it doesn't make any difference: https://unix.stackexchange.com/a/13576